src/commonComponents/linkItem/LinkItem.js

```diff
@@ -0,0 +1,40 @@
+// Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _noop from 'lodash/noop';
+
+function renderLinkItem(props, ref) {
+  const { label, isActive, href, onClick } = props;
+
+  const active = isActive ? 'active' : undefined;
+
+  function handleClick() {
+    return onClick(ref);
```

Comment:
you are getting ref from `menuLink` component and here you are sending it the ref for getting the hash value of anchor tag in the parent component
since the hash value of anchor tag will be same as the href you can send href only no need of ref here

Reply:
I know it's just for practice of refs